plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/_core.py

```diff
@@ -375,3 +281,49 @@ def normalize_key(key: str, str_to_replace: str, str_to_replace_with: str) -> st
     ]
     new_key = str_to_escape.join(updated_splits).replace("\\", "")
     return new_key
+
+
+def create_range_param_using_interval_override(override):
+    key = override.get_key_element()
+    value = override.value()
+    param = {
+        "name": key,
+        "type": "range",
+        "bounds": [value.start, value.end],
+    }
+    return param
+
+
+def create_choice_param_from_choice_override(override):
+    key = override.get_key_element()
+    value = override.value()
+    param = {
+        "name": key,
+        "type": "choice",
+        "values": value.list,
+    }
+    return param
+
+
+def create_choice_param_from_range_override(override):
+    key = override.get_key_element()
+    value = override.value()
+    param = {
+        "name": key,
+        "type": "choice",
+        "values": [val for val in value.range()],
+    }
+
+    return param
+
+
+def create_fixed_param_from_element_override(override):
+    key = override.get_key_element()
```

Comment:
What I don't like about this style is that you separated the check that the value is a value to the usage of the value.

Reply:
`"value": override.value()`, would be better?